client/src/index.html

```diff
@@ -13,6 +13,14 @@
   integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
   crossorigin="anonymous"
 />
+
+<!-- Google font links -->
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Sen:wght@700&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,900&family=Sen:wght@700&display=swap" rel="stylesheet">
```

Comment:
Thank you for adding fonts. I was thinking that it would be good to add a main.css file to define those, instead of putting them in the head. We can then specify what weigh and variants we want from google fonts.

Reply:
All comments are straightforward and really helped me to correct my mistakes and follow good practice. Thank you.